examples/tv-casting-app/tv-casting-common/include/CHIPProjectAppConfig.h

```diff
@@ -55,3 +55,5 @@
 #define CONFIG_BUILD_FOR_HOST_UNIT_TEST 1
 
 #define CHIP_ENABLE_ROTATING_DEVICE_ID 1
+
+#define CHIP_DEVICE_CONFIG_ROTATING_DEVICE_ID_UNIQUE_ID_LENGTH 64
```

Comment:
Should this have some sort of default defined?  Or is the intent that people using rotating device ids should ensure this is defined or get compile errors?

Reply:
The default is already defined here: https://github.com/project-chip/connectedhomeip/blob/6ccbb7c4b8e859343d5a7562986f8162971b6b22/src/include/platform/CHIPDeviceConfig.h#L288-L290